### PR TITLE
Update instance_blood_furnace.cpp

### PR DESCRIPTION
**Pull request by AnonXS**
https://bitbucket.org/Caboon/playcore-official/commits/c2dd48d81d5072c34612108657acf7e7a4f66947

### CODE REVIEW

**Comment by Xadras** [comment]
seems to be a bit of too many empty lines :>

File: src/scripts/scripts/zone/hellfire_citadel/blood_furnace/instance_blood_furnace.cpp [hunk 1]
```diff
@@ -104,19 +104,12 @@ struct instance_blood_furnace : public ScriptedInstance
...
 +                    Sewer2GUID = go->GetGUID(); 
-
 +					
-
 +                    if (Encounter[2] == DONE)
-
 +                        HandleGameObject(Sewer2GUID, 0);
-
 +                }
-
```
Comment (Micha2903) [inline src/scripts/scripts/zone/hellfire_citadel/blood_furnace/instance_blood_furnace.cpp]: break; should be inside case 181766

Comment (AnonXS) [inline src/scripts/scripts/zone/hellfire_citadel/blood_furnace/instance_blood_furnace.cpp]: okay so the function doesnt work like this, then maybe the flaw is at

            case DATA_KELIDANEVENT:
                if (data == DONE)
                {
                    HandleGameObject(Sewer1GUID, 0); 
                    HandleGameObject(Sewer2GUID, 0);

he only handels the Sewer1GUID but doesnt handle Sewer2GUID.

**Comment by danqu** [comment]
@AnonXS  Playcore-official this project can not be used within the vmaps, mmaps, map-tools can not be extracted, I would like to ask how to solve, thank you very much

**Pull request closed without merging**